unix/modos.c

```diff
@@ -228,6 +236,14 @@ STATIC const mp_rom_map_elem_t mp_module_os_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_getenv), MP_ROM_PTR(&mod_os_getenv_obj) },
     { MP_ROM_QSTR(MP_QSTR_mkdir), MP_ROM_PTR(&mod_os_mkdir_obj) },
     { MP_ROM_QSTR(MP_QSTR_ilistdir), MP_ROM_PTR(&mod_os_ilistdir_obj) },
+    #if MICROPY_FSUSERMOUNT
+    { MP_ROM_QSTR(MP_QSTR_vfsmount), MP_ROM_PTR(&fsuser_mount_obj) },
+    { MP_ROM_QSTR(MP_QSTR_vfsumount), MP_ROM_PTR(&fsuser_umount_obj) },
+    { MP_ROM_QSTR(MP_QSTR_vfsmkfs), MP_ROM_PTR(&fsuser_mkfs_obj) },
```

Comment:
But vfs is already an acronym (unlike "list" and "dir") so is harder to read (and it's not familiar like mkdir is).  But I hope that these ugly names are only for unix and baremetal ports can chose to use mount/umount/mkfs :)